test/src/test/java/org/corfudb/infrastructure/ReconfigurationEventHandlerTest.java

```diff
@@ -57,12 +59,18 @@ public void updateLayoutOnFailure() {
         Set<String> failedServers = new HashSet<>();
         failedServers.add(getEndpoint(SERVERS.PORT_2));
 
-        ReconfigurationEventHandler reconfigurationEventHandler = new ReconfigurationEventHandler();
+        // Exclude SERVER 2 from the Quorum of this test layout to prevent it from getting a
+        // falsely positive response from sealMinServerSet() method.
+        addClientRule(corfuRuntime, SERVERS.ENDPOINT_2, new TestRule().always().drop());
+
         IReconfigurationHandlerPolicy failureHandlerPolicy = new ConservativeFailureHandlerPolicy();
-        reconfigurationEventHandler.handleFailure(failureHandlerPolicy,
-                        originalLayout,
-                        corfuRuntime,
-                        failedServers);
+        boolean succeed = ReconfigurationEventHandler.handleFailure(failureHandlerPolicy,
+                originalLayout, corfuRuntime, failedServers);
+
+        clearClientRules(corfuRuntime);
+
+        // Verify that handleFailure correctly returns true.
+        assertTrue(succeed);
```

Comment:
'assertThat' has better readability and exception message.

Reply:
Changed, thanks!